packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2165,6 +2166,29 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     }
   }
 
+  // Sends the current composing rect to the iOS text input plugin via the text
+  // input channel. We need to keep sending the information even if no text is
+  // currently marked, as the information usually lags behind. The text input
+  // plugin needs to estimate the composing rect based on the latest caret rect,
```

Comment:
Can this lag cause jitter when the true information is received? 1 frame swapping between an estimate and true info may be a source of jank.

Reply:
The "true" information does not trigger a UI update of any sort when it's sent to the platform. It will be used as an estimate the next time iOS requests this information.